Messages test arbitrary failed

[One of my tests](https://travis-ci.org/OpenGrok/OpenGrok/builds/175678448) failed. Looks like something is wrong with the test, not with the code.
```log
testApplyMultipleUnique(org.opensolaris.opengrok.configuration.messages.NormalMessageTest)  Time elapsed: 0.008 sec  <<< FAILURE!

java.lang.AssertionError: expected:<1> but was:<2>

	at org.opensolaris.opengrok.configuration.messages.NormalMessageTest.testApplyMultipleUnique(NormalMessageTest.java:117)
```
```log
Results :

Failed tests: 

  NormalMessageTest.testApplyMultipleUnique:117 expected:<1> but was:<2>
```

--- a/test/org/opensolaris/opengrok/configuration/messages/NormalMessageTest.java
+++ b/test/org/opensolaris/opengrok/configuration/messages/NormalMessageTest.java
@@ -103,9 +103,11 @@ public class NormalMessageTest {
     @Test
     public void testApplyMultipleUnique() {
         Message[] m = makeArray(new NormalMessage(), new NormalMessage(), new NormalMessage());
+        Date d = new Date();
 
         for (int i = 0; i < m.length; i++) {
             m[i].addTag("main");
+            m[i].setCreated(d);
         }
 
         Assert.assertEquals(0, env.getMessagesInTheSystem());

